packages/vue-components/src/SearchbarPageItem.vue

```diff
@@ -1,15 +1,13 @@
 <!-- eslint-disable vue/no-v-html -->
 <template>
   <div v-if="item.heading" class="heading">
+    <i class="fa fa-hashtag"></i>
     <div class="heading-text">
-      <span v-html="item.heading.text"></span>
-    </div>
-    <div class="heading-text-items">
-      <small v-html="highlight(item.heading.text, value)"></small>
-      <br />
+      <span v-html="highlight(item.heading.text, value)"></span>
       <small v-for="(keyword, index) in item.keywords" :key="index">
-        <span v-html="highlight(keyword, value)"></span>
-        <br />
+        <div class="heading-text">
```

Comment:
I see you have this div and class "heading-text" to ensure that the keyword starts on a new line. Wonder if you have tried simplifying it to `<div v-html="highlight(keyword, value)"></div>` and do without the class? Seems like it works for most cases. What do you think @weiquu?

Reply:
Thanks I think it works well! Have changed it

Reply:
Hi @weiquu, sorry for going back and forth on this one (the rest of the PR LGTM). On second thought I feel that this update might not be very semantic because small is an inline element while div is a block element, so inline should technically not contain a block.

 I am suggesting to change to the following to avoid possible complication down the line:

```
    <div class="heading-text">
      <span v-html="highlight(item.heading.text, value)"></span>
      <div
        v-for="(keyword, index) in item.keywords"
        :key="index"
        class="keyword-text"
        v-html="highlight(keyword, value)"
      ></div>
    </div>

.keyword-text {
  font-size: smaller;
}
```

(P.S then again, the rule is broken in the dropdown level outside of this, where an inline `<a>` wraps the `<div>`, but things still work fine...) pinging @ang-zeyu @jonahtanjz for advice on this 👁️  
![image](https://user-images.githubusercontent.com/41845017/188251068-e5401cf8-0f9b-4cc8-a2d5-bd0dd6c2b71f.png)

Reply:
> (P.S then again, the rule is broken in the dropdown level outside of this, where an inline <a> wraps the <div>, but things still work fine...) pinging @ang-zeyu @jonahtanjz for advice on this 👁️

This is actually ok (*not* the `<small><div>...` though).

Some links: 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#properties 
https://html.spec.whatwg.org/#transparent-content-models
 to validate the markup - https://validator.w3.org/nu/#textarea 
You can also check out how algolia docsearch structures its html - https://docusaurus.io/ (just using docusaurus as a convenient example)

I am nowhere an expert in this area too 😅😂  but in general also note browsers don't comply with the spec completely; Failing gracefully is as is the case here is often the better option for the end user. https://stackoverflow.com/questions/64821044/how-browser-restructure-the-invalid-html-structure-is-there-any-rules-or-approa

Reply:
SSR hydration assertions also do not check everything, likely just some of the most common ones like `<table>`s that browsers autocorrect aggressively

Reply:
Thanks for the links! Have edited according to @tlylt's suggestion